src/main/scala/is/hail/expr/Parser.scala

```diff
@@ -535,7 +535,8 @@ object Parser extends JavaTokenParsers {
 
   def type_expr: Parser[Type] =
     "Empty" ^^ { _ => TStruct.empty } |
-      "Interval" ^^ { _ => TInterval(GenomeReference.GRCh37) } |
+      ("Interval" ~ "(") ~> identifier <~ ")" ^^ { id => TInterval(GenomeReference.getReference(id)) } |
+      "Interval" ^^ { _ => TInterval(GenomeReference.GRCh37) } | // Backwards compatibility
```

Comment:
This is 0.2 so we don't need to support backward compatibility.  Do we want to support this going forward?  If not, we should remove.  Same with Variant, Locus below.